src/example_exercises/16_color_contour_detector.py

```
@@ -0,0 +1,297 @@
+        cv2.putText(
+            img,
+            f"Offset: ({offset_x}, {offset_y})",
+            (10, 30),
+            cv2.FONT_HERSHEY_SIMPLEX,
+            0.6,
+            (255, 255, 255),
+            2,
+        )
+        cv2.putText(
+            img,
+            f"Yaw:{yaw_velocity} Fwd:{forward_velocity} " f"UpDn:{up_down_velocity}",
```

Ruff (0.15.21)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)